src/api.js

```diff
@@ -0,0 +1,7 @@
+const URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=a2426096&t=';
```

Comment:
```suggestion
const URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=a2426096&t=';
```
Without it your app won't work correctly on github

Reply:
FIxed